boards/shared/Inc/Io/Io_SharedFrequencyOnlyPwmInput.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <stm32f3xx_hal.h>
+#include <stdbool.h>
+
+#include "Io_SharedFrequencyOnlyPwmInputConfig.h"
+
+#ifndef MAX_NUM_OF_FREQ_ONLY_PWM_INPUTS
+#error Missing definition: MAX_NUM_OF_FREQ_ONLY_PWM_INPUTS must be defined in Io_SharedFreqOnlyPwmInputConfig.h.
+#endif
+
+struct FreqOnlyPwmInput;
+
+/**
+ * Allocate and initialize a PWM input using the given (hardware) timer
+ *
+ * @note The given timer must be initialized with:
```

Comment:
Why is this true? If we are just initializing a single PWM input why would we need to initialize two channels?

Reply:
fixed